py/DREAM/Output/SolverNonLinear.py

```diff
@@ -17,6 +17,8 @@ def __init__(self, solverdata=None, output=None):
         """
         super().__init__(solverdata, output)
 
+        if 'solvertime' in solverdata:
+            self.solvertime = np.array([float(x) for x in solverdata['solvertime'][:]])
```

Comment:
You shouldn't have to do this. It should possible to just do ``self.solvertime = solverdata['solvertime'][:]`` since the values should already be ``float``.

Reply:
(Probably there's also a much better way of doing this for ``self.iterations``, by having ``numpy`` do the type conversion... I feel stupid now)